java/build.gradle.kts

```diff
@@ -28,12 +28,12 @@ val javaagentDependency by configurations.creating {
 }
 
 dependencies {
-    compileOnly(platform("io.opentelemetry:opentelemetry-bom:1.25.0"))
-    compileOnly(platform("io.opentelemetry:opentelemetry-bom-alpha:1.25.0-alpha"))
+    compileOnly(platform("io.opentelemetry:opentelemetry-bom:1.26.0"))
+    compileOnly(platform("io.opentelemetry:opentelemetry-bom-alpha:1.26.0-alpha"))
     // Already included in wrapper so compileOnly
     compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi")
     compileOnly("io.opentelemetry:opentelemetry-sdk-extension-aws")
-    javaagentDependency("software.amazon.opentelemetry:aws-opentelemetry-agent:1.26.0")
+    javaagentDependency("software.amazon.opentelemetry:aws-opentelemetry-agent:1.26.0-adot-lambda1")
```

Comment:
What does the 1 entail in the naming of `adot-lambda1`.  Are we considering the idea to continue to make patches and eventually make `adot-lambda2` if needed?

Reply:
Yes, that is right, we might need to document this. The idea with the number is that we can increment the number in case we release one version and need to release a new patch for the same upstream version.

1.26.0-adot-lambda1 -> first set of patches
1.26.0-adot-lambda2 -> first set of patches + more patches.